project/plugins.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8" % "0.11.0")
+addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8" % "0.13.1")
```

Comment:
we need to update the gitter8 version in order to use `maven(com.typesafe.play, sbt-plugin, stable)` in the test